main.go

```diff
@@ -26,7 +26,6 @@ func main() {
 	bPackages := filegenerator.RenderHistoricalBenchmarkResults(dirs)
 	filegenerator.GenerateJSONFiles(bPackages)
 	filegenerator.GenerateTestNameJSONFile(bPackages)
-	filegenerator.GenerateGeometricMeanJSONFile(bPackages, dirs)
 	filegenerator.CopyWWW()
-	filegenerator.PublishToAWS()
+	//filegenerator.PublishToAWS()
```

Comment:
Thought I had pushed up the correction to this, fixed now. I was testing the code locally and didn't want to change the page on aws.